posts/Yoann/2017-11-27-introducing-hibernate-search-jsr352-mass-indexing-job.adoc

```diff
@@ -0,0 +1,158 @@
+= Introducing the Hibernate Search JSR-352 mass indexing job
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "JSR-352" ]
+:awestruct-layout: blog-post
+---
+
+Hibernate Search 5.9 (still in beta) features a new integration to JSR-352 runtimes,
```

Comment:
could we reprhase this to also serve as "teaser" for the `5.9.0.Beta1` release?

Proposal:
`Hibernate Search 5.9 will feature integration with JSR-352 [...] Today we released 5.9.0.Beta1 so you can try it out already!`